packages/ui/src/components/va-menu/VaMenu.stories.ts

```diff
@@ -0,0 +1,171 @@
+import { VaDivider, VaButton, VaDropdown, VaDropdownContent, VaIcon } from '../../components'
+import VaMenuList from './components/VaMenuList.vue'
+import VaMenuItem from './components/VaMenuItem.vue'
+import VaMenuGroup from './components/VaMenuGroup.vue'
+import { VaAvatar } from '../'
+
+export default {
+  title: 'VaMenu',
+  component: VaMenuList,
+}
+
+export const Default = () => ({
+  components: { VaMenuList },
+  data: () => ({
+    options: [
+      { id: '0', text: 'one', value: 'one' },
+      { id: '1', text: 'two', value: 'two' },
+      { id: '2', text: 'three', value: 'three' },
+    ],
+    value: false,
```

Comment:
I think the default for menu should include selected action. As that would be its main usecase.